manifests/bundle.yaml

```diff
@@ -44,18 +47,21 @@ data:
     kind: MetricsExporterConfig
     logLevel: info
     timeout: 10
+    port: 9100
 kind: ConfigMap
 metadata:
   labels:
     app: storageos
-  name: storageos-metrics-exporter-6kd7dt7582
+    app.kubernetes.io/component: metrics-exporter
+  name: storageos-metrics-exporter-24g7bdh4mf
```

Comment:
The default behavior of [kustomize's configMapGenerator](https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#configmapgenerator) is to add a random suffix to the generated configmap name. The motivation looks to be to preserve the existing cm when re-deploying to a new one with `kubectl apply -f ...`.

We can change it though: https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/#generatoroptions

Since the suffix is present in the bundled daemonset, I don't see a harm in the default behavior. Nevertheless, we don't need to preserve the old managed configmaps because the end-user needs to provide his own anyway. So I am now inclined to no-suffix.

Reply:
Got it, thanks for the detailed explanation 👍 
I would remove the suffix too